app/controllers/spotlight/catalog_controller.rb

```diff
@@ -22,10 +22,6 @@ class CatalogController < ::CatalogController
 
     before_action :load_document, only: %i[edit update make_private make_public manifest]
 
-    before_action only: :show do
```

Comment:
I suspect we did this so previously generated applications also got the right document component. I don't know what the right backwards-compatible pattern might be 🤷‍♂️

Reply:
The current implementation breaks any apps (like our exhibits app) that have additional show partials configured. Is there a reason why a previously generated app needs to use the document component? It appeared to me the if the document component wasn't configured that it would fall back to using the default show partial. My thought was it might be better to let previously generated apps opt-in, but it's possible I'm not understanding something.

Reply:
Nope, I think you've probably got it right. I do think this _was_ a nod at backwards compatibility but only works with Blacklight 8 (with https://github.com/projectblacklight/blacklight/commit/6467c5d3631dfd4d127c73e2e5cdf85e1ad6e03f).